content/en/docs/pipelines/installation/overview.md

```diff
@@ -131,21 +151,27 @@ Interfaces
   * Kubeflow Pipelines UI via the **Open Pipelines Dashboard** link in the 
     GCP Console
   * Kubeflow Pipelines SDK in Cloud Notebooks
+  * Kubeflow Pipelines endpoint of your instance is auto-configured for you.
 
 Release Schedule
 : AI Platform Pipelines is available for a chosen set of stable Kubeflow
 Pipelines releases. You will receive updates slightly slower than Kubeflow
 Pipelines Standalone.
 
 Upgrade Support (**Alpha**)
-: In place upgrade is not supported. Reinstall new version to upgrade is supported without official documentation. You can [delete a
-AI Platform Pipelines application](https://cloud.google.com/ai-platform/pipelines/docs/getting-started#clean_up) **WITHOUT** selecting **Delete cluster** checkbox and
-reinstall the newer version in the same cluster and same namespace. Persisted
-artifact and database data are in GCP persistent disks and will be automatically
-picked up during reinstallation.
+: In place upgrade is not supported. Reinstall new version while retaining data
```

Comment:
Potential syntax issues.

Suggestion:

> An in-place upgrade... Reinstalling a new version ...

Reply:
Another suggestion:

To upgrade your AI Platform Pipelines cluster:

1.  [Delete your AI Platform Pipelines instance](https://cloud.google.com/ai-platform/pipelines/docs/getting-started#clean_up) **without** selecting **Delete cluster**.
1.  Reinstall Kubeflow Pipelines from the Google Cloud Marketplace using the same namespace, and Google Kubernetes Engine cluster. Persisted artifacts and database data are stored in persistent disks and will be automatically picked up during reinstallation.

Reply:
Or, per @amygdala's comment, perhaps this section should be removed?